docs/data/base/components/slider/slider.md

```diff
@@ -0,0 +1,60 @@
+---
+product: base
+title: React Slider unstyled component and hook
+components: SliderUnstyled
+githubLabel: 'component: slider'
+waiAria: https://www.w3.org/TR/wai-aria-practices/#slider
+packageName: '@mui/base'
+---
+
+# Slider
+
+<p class="description">Sliders allow users to make selections from a range of values.</p>
+
+Sliders show a range of values along a bar, from which users may select a single value.
```

Comment:
```suggestion
Sliders show a range of values along a bar, from which users may select a single value, or a range.
```

Not sure how to phrase that users can select a range too, for example from the sliders with two thumbs.